collector/pg_stat_statements.go

```diff
@@ -16,6 +16,8 @@ package collector
 import (
 	"context"
 	"database/sql"
+	"fmt"
+	"github.com/alecthomas/kingpin/v2"
```

Comment:
This should be sorted with other `github.com` imports below, not with the standard library imports.